tools/pathifier/.shed.yml

```diff
@@ -0,0 +1,15 @@
+# .shed.yml supporting automatic pushes.
+owner: artbio
+name: pathifier
+description: pathifier
+long_description: |
+  pathifier is a method to reduce dimension of dataset, cop with curse dimensionality
+  and find signatures in patient transcriptome data
```

Comment:
```
  pathifier is a method to reduce dimension of dataset, cop with curse of dimensionality
  and find gene expression signatures in patient transcriptome data
```

Reply:
pathifier is a method to reduce the dimensionality of a dataset